### PR TITLE
Add new white list for bsc#1215378

### PR DESCRIPTION
**Pull request by rfan1**
https://bugzilla.suse.com/show_bug.cgi?id=1215378

- Verification run: https://openqa.suse.de/tests/14224199#step/journal_check/21

### CODE REVIEW

File: data/journal_check/bug_refs.json [hunk 1]
```diff
@@ -515,7 +515,7 @@
         "type": "bug"
     },
     "bsc#1215378": {
-        "description": "(libvirtd|virtqemud).*: error from service: GDBus.Error:org.freedesktop.login1.OperationInProgress: The operation inhibition has been requested for is already running",
+        "description": "(libvirtd|virtqemud).*: error from service: GDBus.Error:org.freedesktop.(login1.OperationInProgress: The operation inhibition has been requested for is already running|systemd1.ShuttingDown: Refusing activation, D-Bus is shutting down)",
```
Comment (jlausuch) [inline data/journal_check/bug_refs.json:518]: why that additional `(` ?

Comment (rfan1) [reply data/journal_check/bug_refs.json:518]: https://openqa.suse.de/tests/14167463#step/journal_check/22

`Apr 29 15:53:25.414099 localhost.localdomain virtqemud[979]: error from service: GDBus.Error:org.freedesktop.systemd1.ShuttingDown: Refusing activation, D-Bus is shutting down.
`

Comment (jlausuch) [reply data/journal_check/bug_refs.json:518]: No, what I mean is you replaced
`GDBus.Error:org.freedesktop.login1.OperationInProgress`
by
`GDBus.Error:org.freedesktop.(login1.OperationInProgress`
in the original error message.

Comment (jlausuch) [reply data/journal_check/bug_refs.json:518]: ok, never mind, I dind't see the closing `)` at the end...

**Review by jlausuch — APPROVED** [review]

**Pull request merged**